src/bidiMapper/domains/browser/BrowserProcessor.ts

```diff
@@ -38,9 +40,20 @@ export class BrowserProcessor {
     return {};
   }
 
-  async createUserContext(): Promise<Browser.CreateUserContextResult> {
+  async createUserContext(
+    params: Record<string, any>
```

Comment:
Consider using something similar to [this TypeScript ](https://www.typescriptlang.org/play?#code/LAKAlgdgLgpgTgMwIYGMYAIDCcZNgITgHsB3AZ3kyOhgA8oAlGARwFcYyp0BvUddAPQAqIX35D0ASQToyABxgowCMDAAmAGnRqw8ohTLooAC13oU1WPXQljMCNpgAjVgHNXkV7I5kw1bboWEBCKUGQAdGLoQgJiOnoUAPIQACIwUKjGAPwAXOhOREQANrgQANxiwqIg-NHoAArEtACe3nAAbvBavgC2YEVIcEZERnbo1BhySGQUasPoALQLck3NCxQd8FExYitELQDK8J1wubJQcJ4VNYIi2w2r+c1TM+hFulDdYH0DQ1AjJgwE3QL1m8yWexaCycz2mZAW70421iN0hzXwsJmABkPmdOJcIK5rvwqvcAIIOIhyKB+CBIIpvD7jGRES4eCCGf7oVxwJDQdCsCDvPqwOYoYgzBassDs9CoNCvf6RG51eqDMLM0YYACqDCx40Bf2MfNG+gwQU4YCgrBphPGbMgcpw6BlEFZ6mVtR2N2l7LIAHUrcZtRAwCcyPSAHLpEisgDWZJQCrIeIungA2gBda4AX1AoDoclZXAtXFchVcjRgKlo6AAvOgAOTloiuHKNuWGUvXAu0ItwLhQZ4YAAqRAA4hWAGr09hkAA8I4AfPWeGJ0wBpF0OOMwZpEGQjzvoLd0WAQNSGfGedBZdAAAwAJNwhwoD9yK1Waznnxuc-f0DyEITkzPIR03bNQBzHsQFfDBYE4Vcx0nVsZyKOd52wXACGIchKEsM8mDYDgoCXGDSyMPIEK4BteBudNmwrHJ4iLJJUnSTJG1Aow4HYDQoJgqAYNAIA)?